XX.md

```diff
@@ -0,0 +1,108 @@
+# NIP-XX
+
+## Static Payment Addresses
+
+`draft` `optional`
```

Comment:
Can you change the format to this, like other NIPs. It's good to have a standard format for NIPs viewers.

```suggestion
NIP-XX
======

Static Payment Addresses
-----------------------

`draft` `optional`
```